src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Windows.targets

```
@@ -27,6 +27,7 @@ Copyright (c) .NET Foundation. All rights reserved.
   <PropertyGroup Condition="'$(TargetPlatformIdentifier)' == 'windows' and '$(TargetPlatformVersion)' == ''" >
     <TargetPlatformVersion>7.0</TargetPlatformVersion>
     <!-- Automatic generation of TargetPlatformMoniker and TargetPlatformDisplayName have already been evaluated so we need to hardcode these values. -->
+    <SupportedOSPlatform>7.0</SupportedOSPlatform>
     <TargetPlatformMoniker>Windows,Version=7.0</TargetPlatformMoniker>
     <TargetPlatformDisplayName>Windows 7.0</TargetPlatformDisplayName>
```

Any reason we don't use `$(TargetPlatformVersion)` that we defined right above? Seems odd to duplicate the version number.

```suggestion
    <SupportedOSPlatform>$(TargetPlatformVersion)</SupportedOSPlatform>
    <TargetPlatformMoniker>Windows,Version=$(TargetPlatformVersion)</TargetPlatformMoniker>
    <TargetPlatformDisplayName>Windows $(TargetPlatformVersion)</TargetPlatformDisplayName>
```